### PR TITLE
Minor fixes in a website

### PR DESCRIPTION
**Pull request by cp-shruti-s**
1. Remove "As you read above" in CTA section
2. Add youtube icon in footer.
3. Add new line from "Canopas is open source!" in contribute section 
4. Add new line from "Our blog hosted..." in blog section
5. Remove portfolio link in header section

### CODE REVIEW

File: vue-frontend/src/components/home-new/ContributeSection.vue [hunk 1]
```diff
@@ -5,9 +5,11 @@
       <div class="border-text v2-header-text">here to</div>
       <div class="v2-title-3-text">
         <span class="v2-header-text pe-2">contribute</span>
-        and that starts with the site itself, canopas is open ource! We also
-        have several projects that are widely recognized in the open source
-        community.
+        and that starts with the site itself,
+      </div>
+      <div class="v2-title-3-text mt-1">
+        canopas is open ource! We also have several projects that are widely
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/ContributeSection.vue:11]: Fix `ource`

File: vue-frontend/src/components/partials/ScreenHeaderV2.vue [hunk 2]
```diff
@@ -66,37 +67,43 @@ export default {
           url: "/",
           className: "nav-link is-animation-tab",
           target: false,
+          isActive: true,
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/partials/ScreenHeaderV2.vue:70]: Why all are active?

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/partials/ScreenHeaderV2.vue (hunk 2 above)
Comment (cp-shruti-s) [reply vue-frontend/src/components/partials/ScreenHeaderV2.vue:70]: Because remove only portfolio link

Comment (cp-sumi-k) [reply vue-frontend/src/components/partials/ScreenHeaderV2.vue:70]: Why do we need to add extra vars for that? Can't we comment portfolio link?

Comment (cp-shruti-s) [reply vue-frontend/src/components/partials/ScreenHeaderV2.vue:70]: I think the flag will be better than the commented code

Comment (cp-sumi-k) [reply vue-frontend/src/components/partials/ScreenHeaderV2.vue:70]: It's fine to comment if we will need it.  Just add to do comment.

Comment (cp-shruti-s) [reply vue-frontend/src/components/partials/ScreenHeaderV2.vue:70]: okay

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**